flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/ContinuousFileMonitoringFunction.java

```diff
@@ -282,7 +282,7 @@ private void monitorDirAndForwardSplits(FileSystem fs,
 	 * Returns the paths of the files not yet processed.
 	 * @param fileSystem The filesystem where the monitored directory resides.
 	 */
-	private Map<Path, FileStatus> listEligibleFiles(FileSystem fileSystem) throws IOException {
+	private Map<Path, FileStatus> listEligibleFiles(FileSystem fileSystem, String path) throws IOException {
```

Comment:
I would suggest passing a `Path` here. It is always a safer option to rely on the this class than on strings.